examples/fabric-admin/scripts/run_fabric_sink.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Default paths
+FABRIC_ADMIN_PATH="./fabric-admin"
+FABRIC_ADMIN_LOG="/tmp/fabric_admin.log"
+
+# Detect platform and set default paths
+if [[ $(uname -m) == "x86_64" ]]; then
+    FABRIC_ADMIN_PATH="out/debug/standalone/fabric-admin"
```

Comment:
could we use some file finding logic? For example I compile with build_examples.py so the path is never `debug/standalone`. I would suggest some form of iteration of predefined paths and the first one wins. like:

```sh
find_fabric_admin() {
   local choices=(
       "./fabric-admin"
       "out/debug/standalone/fabric-admin"
       "out/linux-x64-fabric-admin/fabric-admin"
       "out/darwin-arm64-fabric-admin/fabric-admin"
   )
   
   for path in "${choices[@]}"; do
       if [[ -e "$path" ]]; then
          echo "$path"
          return 0
      fi
   done
  return 1
}

FABRIC_ADMIN_PATH=$(find_fabric_admin)
if [[ $? -ne 0 ]]; then
   >&2 echo "Could not find the fabric-admin binary"
   exit 1
fi
echo "PATH IS: $FABRIC_ADMIN_PATH"
```
Potentially allow an argument to specify the path to use.